Enable copying podcast title and host #7809

GiannisMakris opened the pull request:
Closes #7809

### Description

Added **android:textIsSelectable** in 
`<TextView
                    android:id="@+id/txtvTitle"
                    android:textIsSelectable="true" 
... />`

located at _feeditem_fragment.xml_. This makes episode's title selectable inside **ItemFragment**.

Also added **android:textIsSelectable** in 
`<TextView
                android:id="@+id/txtvTitle"
                android:textIsSelectable="true"
                ... />`

and 

`<TextView
                android:id="@+id/txtvAuthor"
                android:textIsSelectable="true"
                ... />`

located at _feeditemlist_header.xml_. This makes podcasts title and author selectable inside **FeedInfoFragment**.
 
### Checklist
<!-- 
  To help us keep the issue tracker clean and work as efficient as possible,
  please make sure that you have done all of the following.
  You can tick the boxes below by placing an x inside the brackets like this: [x]
-->
- [x] I have read the contribution guidelines: https://github.com/AntennaPod/AntennaPod/blob/develop/CONTRIBUTING.md#submit-a-pull-request
- [x] I have performed a self-review of my code
- [x] I have run the automated code checks using `./gradlew checkstyle spotbugsPlayDebug spotbugsDebug :app:lintPlayDebug`
- [x] My code follows the style guidelines of the AntennaPod project: https://github.com/AntennaPod/AntennaPod/wiki/Code-style
- [x] I have mentioned the corresponding issue and the relevant keyword (e.g., "Closes: #xy") in the description (see https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [x] If it is a core feature, I have added automated tests

ByteHamster reviewed (commented):
Thanks for your pull request. Currently the CI still fails. See my comment below.

GiannisMakris commented:
OK it seems that setting _textIsSelectable_ in a TextView breaks _setEllipsize()_ on the TextView. I removed the xml attributes and added longClickListeners on the corresponding TextViews.

ByteHamster commented:
Sounds good, could you push the change please? :) On the podcast info page, we have a button to copy the url. There we show a snackbar message. Could you please show the same message when copying the title?

ByteHamster commented:
Thanks! Will be released in AntennaPod 3.9.0. In case you didn't know yet, we are having a monthly call to meet up with other AntennaPod volunteers. You are warmly invited in case you want to join: https://antennapod.org/events/community-meeting :)
